src/webgl/interaction.js

```diff
@@ -371,14 +366,13 @@ p5.prototype.debugMode = function(...args) {
  *   if (keyIsDown(32)) {
  *     noDebugMode();
  *   }
+ *   describe(`a 3D box is centered on a grid in a 3D sketch.
+ *     an icon indicates the direction of each axis:
+ *     a red line points +X, a green line +Y, and a blue line +Z.
+ *     the grid and icon disappear when the spacebar is pressed.`);
```

Comment:
can we make it more succinct?

Reply:
Same response as my [first comment](https://github.com/processing/p5.js/pull/5599#discussion_r815326608).